rt_utils/ds_helper.py

```diff
@@ -65,9 +65,9 @@ def add_sequence_lists_to_ds(ds: FileDataset):
 def add_study_and_series_information(ds: FileDataset, series_data):
     reference_ds = series_data[0] # All elements in series should have the same data
     ds.StudyDate = reference_ds.StudyDate
```

Comment:
```suggestion
    ds.StudyDate = getattr(reference_ds, 'StudyDate', '')
```

Reply:
Hi @awtkns 

I did consider supporting missing StudyDate and StudyTime. But those are actually required by the DICOM standard. So any datasets which don't contain them would not be valid DICOM. Do you think we should support it anyway?

Reply:
That's tricky, to be honest I am not sure. It might be best to stick with it then as it is required.  On the flipside having an empty default would probably cause no harm and could potentially save from issues being reported.

Reply:
As @pchlap says, those fields are actually required by the DICOM standard. We could support it as shown in the suggested change but maybe we should print a message saying that the field does not exist so is being forced to be blank.

what do you think @pchlap and @awtkns ?

Reply:
I don't think printing a warning is the best solution.  I am going to merge and release as is. We can also change it later if it proves to be an issue.